app/controllers/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
```

Comment:
This is looking for  a create method on `Users_controller`
![Screenshot 2019-08-30 at 4 52 27 PM](https://user-images.githubusercontent.com/10922508/64037704-32b32680-cb4e-11e9-8b0c-4fdb85ad7fba.png)

Reply:
fixed by add create method, add users index .